README.md

```diff
@@ -1,24 +1,161 @@
-# README
```

Comment:
**[MANDATORY REQUIREMENTS]**

- [ ] Kindly check the inline comments on the frontend pull request 🤔

**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM Done. Thanks